lib/commands/skill/add-locales/helper.js

```diff
@@ -0,0 +1,148 @@
+const fs = require('fs-extra');
+const path = require('path');
+const R = require('ramda');
+const async = require('async');
+
+const httpClient = require('@src/clients/http-client');
+const SkillMetadataController = require('@src/controllers/skill-metadata-controller');
+const CliError = require('@src/exceptions/cli-error');
+const ResourcesConfig = require('@src/model/resources-config');
+const Manifest = require('@src/model/manifest');
+const stringUtils = require('@src/utils/string-utils');
+const CONSTANTS = require('@src/utils/constants');
+
+module.exports = {
+    initiateModels,
+    addLocales
+};
+
+/**
+ * Start with loading the project Models global instances.
+ * @param {String} profile
+ */
+function initiateModels(profile) {
```

Comment:
please consider using ES6 class for helper

Reply:
i still think the consistency is more important since it's just different way to put it. How to abstract the Helper logic for each explicit command and even remove this file seems more important.